### DISCUSSION TITLE
Check use of `SharedArrayBuffer`

### DISCUSSION THREAD

**Discussion by michaelweinold** [jupyterlite/jupyterlite#1451] [Q&A]
Is there a quick way to check if [the new file access via `SharedArrayBuffer`](https://jupyterlite.readthedocs.io/en/latest/howto/content/python.html) (rather than a service worker) works?

All I could think of is the absence of the "Network" tab and the logged file sync of the service worker there:

<img width="1242" alt="Screenshot 2024-08-01 at 18 34 06" src="https://github.com/user-attachments/assets/f805c7cf-2be5-4e10-a599-cf30b1b4901d">

**Comment by jtpio** [comment]
Maybe the kernels with support for `SharedArrayBuffer` should print a message in the dev tools console to indicate whether they use `SharedArrayBuffer` or the Service Manager.

For example here for the Pyodide kernel: https://github.com/jupyterlite/pyodide-kernel/blob/f16cba8dcecd26c23361f8a28c708c4006812f6c/packages/pyodide-kernel-extension/src/index.ts#L82-L91
